servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpHeaderNames.java

```diff
@@ -180,70 +233,104 @@
     public static final CharSequence FORWARDED = newAsciiString("forwarded");
```

Comment:
While you are here, could you align javadoc for this header to follow a consistent format:

```
{@code "forwarded"} is a header field that ...
...

@see <a href="https://tools.ietf.org/html/rfc7239#section-4">RFC7239, section 4</a>
```

Please, also adjust `X_FORWARDED_FOR`, `X_FORWARDED_HOST`, and `X_FORWARDED_PROTO` constant. to use the same format.